xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/BitfinexResilience.java

```diff
@@ -0,0 +1,27 @@
+package org.knowm.xchange.bitfinex;
+
+import io.github.resilience4j.ratelimiter.RateLimiterConfig;
+import java.time.Duration;
+import org.knowm.xchange.client.ResilienceRegistries;
+import org.knowm.xchange.exceptions.RateLimitExceededException;
+
+public final class BitfinexResilience {
+
+  public static final String BITFINEX_RATE_LIMITER = "bitfinexRateLimiter";
+
+  private BitfinexResilience() {}
+
+  public static ResilienceRegistries createRegistries() {
+    ResilienceRegistries registries = new ResilienceRegistries();
+    registries
+        .rateLimiters()
+        .rateLimiter(
+            BITFINEX_RATE_LIMITER,
+            RateLimiterConfig.from(registries.rateLimiters().getDefaultConfig())
+                .limitRefreshPeriod(Duration.ofMinutes(1))
+                .limitForPeriod(90)
+                .drainPermissionsOnResult(e -> RateLimitExceededException.class.isInstance(e))
```

Comment:
did you test this? I am not sure that drainPermissionsOnResult returns an exception direct?

Reply:
your suspicion is correct - this will never work as e is of class `Either`, it will never be of class `Exception`

Reply:
also Bitfinex will never throw an `RateLimitExceededException` directly

Reply:
At the moment, I will delete it until a better understanding